swift/ql/lib/codeql/swift/frameworks/StandardLibrary/Set.qll

```diff
@@ -0,0 +1,15 @@
+private import codeql.swift.dataflow.ExternalFlow
+
+/**
+ * A model for `Set` and related class members that permit data flow.
+ */
+private class SetSummaries extends SummaryModelCsv {
+  override predicate row(string row) {
+    row =
+      [
+        ";Set;true;insert(_:);;;Argument[-1].SetElement;ReturnValue.TupleElement[1];value",
+        ";Set;true;insert(_:);;;Argument[0];Argument[-1].SetElement;value",
+        ";Set;true;randomElement();;;Argument[-1].SetElement;ReturnValue;value"
```

Comment:
I think this should be to `ReturnValue.OptionalSome`, now that we model enum content.